src/components/NftTile/NftTile.tsx

```diff
@@ -0,0 +1,144 @@
+import React, { useEffect, useMemo, useState } from 'react'
+
+import { NftTileDetails } from '@/components/NftTileDetails'
+import { Pill, PillGroup } from '@/components/Pill'
+import { SvgActionAuction, SvgActionBuyNow, SvgActionNotForSale, SvgActionShow } from '@/components/_icons'
+import { VideoThumbnail, VideoThumbnailProps } from '@/components/_video/VideoThumbnail'
+import { formatNumberShort } from '@/utils/number'
+import { formatDurationShort } from '@/utils/time'
+
+import { Container } from './NftTile.styles'
+
+import Timeout = NodeJS.Timeout
+
+export type Member = {
+  assetUrl?: string | null
+  name: string
+}
+
+export type NftTileProps = {
+  auction: 'none' | 'minBid' | 'topBid' | 'waiting'
+  buyNow?: boolean
+  thumbnail: VideoThumbnailProps
+  loading?: boolean
+  title: string
+  owner: Member
+  creator: Member
+  timer?: string
+  duration?: number | null
+  views?: number | null
+  bid: number
+  minBid?: number
+  topBid?: number
+  timeLeft?: number
+  role: 'owner' | 'viewer'
+  fullWidth?: boolean
+}
+
+export const NftTile: React.FC<NftTileProps> = ({
+  auction,
+  buyNow,
+  thumbnail,
+  loading,
+  title,
+  creator,
+  owner,
+  duration,
+  views,
+  bid,
+  minBid,
+  topBid,
+  timeLeft,
+  role,
+  fullWidth,
+}) => {
+  const [calculatedTimeLeft, setCalculatedTimeLeft] = useState(0)
+
+  useEffect(() => {
+    let timeLeftInterval: Timeout
+    if (timeLeft) {
+      setCalculatedTimeLeft(timeLeft)
+      timeLeftInterval = setInterval(() => {
+        setCalculatedTimeLeft((prevState) => {
+          if (prevState <= 1) {
+            clearInterval(timeLeftInterval)
+          }
+          return --prevState
+        })
+      }, 1000)
+    }
+
+    return () => {
+      clearInterval(timeLeftInterval)
+    }
+  }, [timeLeft])
+
+  const getBottomLeft = useMemo(() => {
+    switch (auction) {
+      case 'none':
+        return <Pill icon={buyNow ? <SvgActionBuyNow /> : <SvgActionNotForSale />} size="medium" />
```

Comment:
We should use variant `overlay` for all the pills
![image](https://user-images.githubusercontent.com/51168865/149757512-3773ddc9-b833-46ae-a06e-02ac8b785431.png)